packages/components/button/button.jsdom.test.mjs

```diff
@@ -1,11 +1,59 @@
+import { getByRole } from '@testing-library/dom'
+
 import initButton from './button.js'
 
-describe('mis-instantiation', () => {
-  it('does not prevent further JavaScript from running', () => {
-    expect(() => {
-      // `undefined` simulates the element being missing,
-      // from an unchecked `document.querySelector` for example
-      initButton(undefined)
-    }).not.toThrow()
+describe('Button', () => {
+  /** @type {HTMLButtonElement} */
+  let button
+
+  beforeEach(() => {
+    document.body.innerHTML = `
+      <button class="nhsuk-button" data-module="nhsuk-button" type="submit">
+        Save and continue
+      </button>
+    `
+
+    button = getByRole(document.body, 'button')
+
+    jest.spyOn(button, 'addEventListener')
+  })
+
+  describe('Exports', () => {
+    it('should export init function', () => {
+      expect(initButton).toBeInstanceOf(Function)
```

Comment:
this kind of test seems a bit unnecessary - if I was to export something else instead I would expect the tests below to fail anyway.